lib/rubocop-aha/version.rb

```diff
@@ -1,5 +1,5 @@
 module RuboCop
   module Aha
-    VERSION = '0.6.1'.freeze
+    VERSION = '0.6.3'.freeze
```

Comment:
The other PR has 0.6.4 by the way.

Reply:
Yeah that was intentional.  Each change requires a version bump. :)

I did it this way so we could merge them in order and hopefully the intent would make sense.